.coderabbit.yaml

```
@@ -0,0 +1,86 @@
+    - "**/__pycache__/**"
+    - "**/dist/**"
+    - "**/build/**"
+
+# Code generation settings
+code_generation:
+  enabled: true
+  language_instructions: |
+    - Generate code that follows project conventions
+    - Include proper error handling
+    - Add comments for complex logic
+    - Follow testing patterns established in the project 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)